### PR TITLE
Feature/add specs to readme

### PR DESCRIPTION
**Pull request by mjkrause**
* added specs to README, incl. flow diagram and new Lambda endpoint
* added try / except constructs to handle HTTP reponses
* redeployed AWS Lambda and tested it using curl
* added two more parameters to `.chalice/config.json`
* updated `requirements.txt`
* `ManifestIO.import_tsv_to_fc` not returns only the status code if all responses were OK, otherwise HTTP error messages are send to stdout

primary reviewer @coverbeck

### CODE REVIEW

File: chalicelib/utils.py [hunk 1]
```diff
@@ -45,7 +46,7 @@ def workspace_exists(self):
         headers = dict(Accept='application/json',
                        Authorization=self.auth)
         r = requests.get(self.url, headers=headers)
-        if r.status_code == 200:
+        if 200 <= r.status_code < 300:
             return True
         else:
             return False
```
Comment (coverbeck) [inline chalicelib/utils.py]: You should return the status code and pass that back on to the client.

File: chalicelib/utils.py [hunk 2]
```diff
@@ -62,17 +63,12 @@ def standup_workspace(self):
...
-        elif r.status_code == 401:
-            print("not authorized")
-        else:
-            print("not sure what's happening")
+        try:
+            r = requests.post(url, json=payload, headers=headers)
+            r.raise_for_status()
+        except requests.exceptions.HTTPError as err:
+            print(err)
+            sys.exit(1)
```
Comment (coverbeck) [inline chalicelib/utils.py]: You should not do an exit, you should return the status code to the client

File: chalicelib/utils.py [hunk 3]
```diff
@@ -107,8 +103,12 @@ def import_tsv_to_fc(self):
...
         participant_buf.seek(0)
         files = {'entities': participant_buf}
-        r1 = requests.post(url, files=files, headers=headers)
-
+        try:
+            response = requests.post(url, files=files, headers=headers)
+            response.raise_for_status()
+        except requests.exceptions.HTTPError as err:
+            print(err)
+            sys.exit(1)
```
Comment (coverbeck) [inline chalicelib/utils.py]: Same as above; you should return status code to client.

File: chalicelib/utils.py [hunk 4]
```diff
@@ -120,9 +120,14 @@ def import_tsv_to_fc(self):
...
-        r2 = requests.post(url, files=files, headers=headers)
-        return {"participant": r1.status_code,
-                "sample": r2.status_code}
+        try:
+            response = requests.post(url, files=files, headers=headers)
+            response.raise_for_status()
+        except requests.exceptions.HTTPError as err:
+            print(err)
+            sys.exit(1)
+
```
Comment (coverbeck) [inline chalicelib/utils.py]: Should return status code

**Review by coverbeck — CHANGES_REQUESTED** [review]

**Comment by coverbeck** [comment]
Is this PR still active? Seems like some of it was already done, e.g., removal of pandas.

**Comment by mjkrause** [comment]
I opened a [PR](https://github.com/DataBiosphere/bagit-firecloud-lambda/pull/11) that addresses your point.

**Pull request closed without merging**